bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-en.html

```diff
@@ -0,0 +1,114 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button, fxa_cta_link with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - More privacy, more protections.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_70') }}
+{% endblock %}
+
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew70' %}
+{% set _utm_campaign = 'whatsnew70' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('logos/firefox/quantum/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success show-firefox-desktop-70">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-box-emphasis">
+        <img class="c-emphasis-box-logo" src="{{ static('img/logos/firefox/logo-monitor.svg') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('Have you checked out Firefox Monitor?') }}
+          <span class="show-fxa-supported-signed-in">{{ _('It’s included with your Firefox account.') }}</span>
```

Comment:
@craigcook the line "It's included with your Firefox account." is being shown to all fx users. Other places I use `show-fxa-supported-signed-in` this is not the case, is there something about this nesting that is voiding that css hook?

![image](https://user-images.githubusercontent.com/30483988/66509508-12995000-ea88-11e9-9116-967aed65ea1a.png)

Reply:
Added a comment on the specific line. There's a more specific rule setting this to display.

`.mzp-c-box-emphasis .c-emphasis-box-title span {
    display: inline;
}`